src/index.html

```diff
@@ -7,6 +7,53 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="wrapper">
+      <div class="calendar">
+        <div class="days-names">
+          <div class="days-names__name">Mon</div>
+          <div class="days-names__name">Tue</div>
+          <div class="days-names__name">Wed</div>
+          <div class="days-names__name">Thu</div>
+          <div class="days-names__name">Fri</div>
+          <div class="days-names__name">Sat</div>
+          <div class="days-names__name">Sun</div>
+        </div>
+        <div class="calendar calendar--month-length--30">
+          <div class="calendar__day calendar__day--start-day_sun"></div>
```

Comment:
start-day should be a modifier for the calendar block
```suggestion
        <div class="calendar calendar--month-length--30 calendar--start-day_sun">
          <div class="calendar__day"></div>
```